crates/sdk/src/api/opt/config.rs

```diff
@@ -92,6 +92,11 @@ impl Config {
 		self
 	}
 
+	/// Get the capabilities for the database
+	pub fn get_capabilities(&self) -> &CoreCapabilities {
```

Comment:
We are leaking `CoreCapabilities` here, which is not part of the public API.

Reply:
As discussed, made it `#[doc(hidden)]` in [4303cec](https://github.com/surrealdb/surrealdb/pull/5300/commits/4303cecf60de11f13bdbd99a2cb3e3aee806fce8)

Reply:
Got rid of the method entirely in [3d8018f](https://github.com/surrealdb/surrealdb/pull/5300/commits/3d8018fa0bfc7d8a68a9b254293ddca019988072)